src/main/java/com/avengers/gamera/service/GameService.java

```diff
@@ -39,6 +40,9 @@ public class GameService {
     public GameGetDto createGame(GamePostDto gamePostDto) {
         isExist(gamePostDto.getName());
 
+        String platform=Optional.ofNullable(gamePostDto.getPlatform()).orElse("unknown platform");
```

Comment:
为什么改这个?

Reply:
我想把它转小写，但是我怕它空指针

Reply:
哦哦 你是想存入的时候 转换成小写是吧  
之前是考虑这里用enum来限定的 除了枚举外的都都直接返回404
不过我们现在前端没有game的写入界面 可以先不管的.  
api的话 platform本来就可以为空的 要不存一个空string?  
```suggestion
        String platform=Optional.ofNullable(gamePostDto.getPlatform()).orElse("");
```
这个能写入吗?

Reply:
而且这里是拼接值  'pc,xbox'  这样的  所以本来就比较随意  就只是为了查询而已 所以真要严格 还得一个个检查 然后自己拼起来